.vscode/settings.json

```diff
@@ -74,5 +74,7 @@
   },
```

Comment:
We cant modify files outside of the java directory, if you want changes outside of java, make them on the main branch